### PR TITLE
feat (size-unit): added VW as a choice among width units

### LINKED ISSUE #2735: Add VW unit
**Is your feature request related to a problem? Please describe.**
From user: "Can you please ask your Team if we can have also just not %,EM,REM OR VH, we need also VW for the elements.”

**Describe the solution you'd like**
Add a VW unit

**Describe alternatives you've considered**
N/A

### PR DESCRIPTION
**Pull request by prconcepcion**
fixes #2735

### CODE REVIEW

**Review by bfintal — CHANGES_REQUESTED** [review]
I don't think these are the only units that can have `vw` and `vh`. If you do a regex search across all JS files, you'll find a number of possible options that can have these new units. Do this search:

```
\[ '(px|%)',
```

**Comment by andeng1106** [comment]
@prconcepcion Remaining issue/s:
- Card block - Width is added for Card Image which originally isn't there. Is this intended? @bfintal 
   - Before, Height is the only setting for the Card Image. 
   
Screenshot 2023-07-25 at 2 42 45 PM

- Card block - Height is now missing
Screenshot 2023-07-25 at 2 43 19 PM
   - Now it's applying like this (width) vs. height:
    https://github.com/gambitph/Stackable/assets/103395655/ef0c3f71-d3ae-4ead-863c-430a1b51ecb8

- Carousel block - Arrows > Button Width doesn't have a vw option
Screenshot 2023-07-25 at 2 47 41 PM

**Comment by andeng1106** [comment]
@prconcepcion Cannot select vw on Image resizer:

https://github.com/gambitph/Stackable/assets/103395655/13dce078-5b03-4779-b459-a534269a6a0d

File: src/block/card/edit.js [hunk 1]
```diff
@@ -164,9 +164,9 @@ const Edit = props => {
 						heightUnits={ heightUnit }
 						defaultWidth={ 250 }
 						width={ blockStyle !== 'horizontal' ? 100 : undefined }
-						widthUnit={ blockStyle !== 'horizontal' ? '%' : 'px' }
+						widthUnit={ blockStyle !== 'horizontal' ? '%' : props.attributes.imageWidthUnit }
 						height={ blockStyle !== 'horizontal' ? undefined : 100 }
-						heightUnit={ blockStyle !== 'horizontal' ? 'px' : '%' }
+						heightUnit={ blockStyle !== 'horizontal' ? props.attributes.imageHeightUnit : '%' }
```
Comment (bfintal) [inline src/block/card/edit.js:169]: There needs to be a fallback here just incase the attribute is not available

**Review by bfintal — CHANGES_REQUESTED** [review]

**Comment by andeng1106** [comment]
Remaining issue:
When adding a new Card block > Horizontal layout, it's not Horizontal in editor. But upon saving page and viewing frontend, it's correct.

https://github.com/gambitph/Stackable/assets/103395655/c1f9cea4-1881-4057-b315-71c723611b8d

@prconcepcion

**Pull request merged**